### PR TITLE
Add dolo.

### PR DESCRIPTION
**Pull request by albop**
Checklist
- [ x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x ] Source is from official source
- [x ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x ] If static libraries are linked in, the license of the static library is packaged.
- [ x] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hmaarrfk** [comment]
lgtm @conda-forge/staged-recipes a friendly ping

File: recipes/dolo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - poetry
+  run:
+    - dolang >=0.0.8
+    - interpolation >=2.1.4
+    - ipython >=7.13.0
+    - matplotlib >=3.0
```
Comment (xhochy) [inline recipes/dolo/meta.yaml:26]: Depend on `-base` if you don't explicitly depend on the Qt backend of matplotlib.
```suggestion
    - matplotlib-base >=3.0
```

Comment (albop) [reply recipes/dolo/meta.yaml:26]: Thank you @xhochy

**Comment by albop** [comment]
I just replaced dependence on matplotlib by matplotlib.base and loosened a bit version number so packages in latest Anaconda don't need updating.

**Pull request merged**